python/tvm/relay/build_module.py

```diff
@@ -208,6 +249,61 @@ def build(mod, target=None, target_host=None, params=None):
     return graph_json, mod, params
 
 
+def optimize(mod, target=None, params=None):
+    """Helper function that builds a Relay function to run on TVM graph
+    runtime.
+
+    Parameters
+    ----------
+    mod : relay.Module
+        The module to build. Using relay.Function is deprecated.
+
+    target : str, :any:`tvm.target.Target`, or dict of str(i.e. device/context
+    name) to str/tvm.target.Target, optional
+        For heterogeneous compilation, it is a dictionary indicating context to
+        target mapping. For homogeneous compilation, it is a build target.
+
+    params : dict of str to NDArray
+        Input parameters to the graph that do not change
+        during inference time. Used for constant folding.
+
+    Returns
+    -------
+    graph_json : str
+        The json string that can be accepted by graph runtime.
```

Comment:
cpp `optimize` does not call `graph_codegen->Codegen`, I don't think it will return valid json, we should remove it since it's not used either.

Reply:
You are right. Changed as suggested.